evaporate.js

```diff
@@ -597,7 +597,6 @@
                         setStatus(COMPLETE);
                     } else {
                         l.d('headObject not found on S3.');
-                        me.name = awsKey;
```

Comment:
Yeah, I think this is unrelated to the IE issue but it just highlighted it because the ETag check failed. me.name is the filename is it not? I think that's why it was a problem as our awsKey also contained a folder name. I could do some deeper digging if you'd like. This fixed the first part of the IE issue though, previously the ETag wouldn't match (due to XML issue) and then the retry of the upload would also fail. Removing this line meant the upload worked.